Makefile

```diff
@@ -209,7 +209,7 @@ push-operator-image: operator-image
 	docker push ${OPERATOR_IMAGE_PATH}
 
 microk8s-operator-update: operator-image
-	docker save ${OPERATOR_IMAGE_PATH} | microk8s.ctr -n k8s.io image import -
+	docker save ${OPERATOR_IMAGE_PATH} | microk8s.ctr --namespac k8s.io image import -
```

Comment:
Why this change?
Does it have a typo?

Reply:
Recently, I got `Cannot use two forms of the same flag: n namespace` error.
Ops, a missing e..